src/sql/run.py

```diff
@@ -11,6 +11,7 @@
 import sqlparse
 import sql.connection
 from .column_guesser import ColumnGuesserMixin
+import traceback
```

Comment:
Our CI/CD container doesn't have `traceback` module installed, that's major reason why this got failed

1. Actually it's better the remove the print/traceback thing on remote branch, since it might cause some unexpected fails like this

2. If you need to install the new package in our source code, add it in `install_requires` in `setup.py`